content/README.md

```diff
@@ -0,0 +1,67 @@
+<content>
+
+# README
```

Comment:
Skrevet en README som kanskje gjør det enklere for Christer å opprette nye markdown contents. Kommenter om noe kan forklares bedre @kstigen @vikre . En kan se den på github ellers kan det også ses via: https://developer.test.digitallibrary.io/README